app/tests/spec/lib/app-start.js

```diff
@@ -37,6 +37,7 @@ function (chai, sinon, AppStart, Session, Constants, p, Url, OAuthErrors,
   'use strict';
 
   /*global describe, beforeEach, it*/
```

Comment:
@TDA: IIUC, since we're setting an ESLint `env` of "mocha", we probably don't need to declare these inline `global` vars anymore.

You may want to do a quick `grep` and see if we can remove the comments throughout the */tests/* directories.